tools/build_variables.bzl

```diff
@@ -485,6 +485,7 @@ torch_mobile_core = [
     # This should not be needed eventually.
     # TODO: Remove this dependency
     "torch/csrc/jit/backends/backend_debug_info.cpp",
+    "torch/csrc/jit/mobile/flatbuffer_loader.cpp",
```

Comment:
added comments in Phabricator as it indirectly touches an internal file

Reply:
I think it's used for OSS now. The change to use it as source of truth hasn't landed yet. 

Aside from it, if it's used by both jit and mobile, can it be added to `core_sources_common`, instead of adding it to both `torch_mobile_core` and `libtorch_extra_sources`?

Reply:
@pavithranrao the internal one is not for production in general, but to be safe let's run the BSB to make sure not size regress is introduced. 

@cccclai It's a WIP PR that is not immediately to be used in production. It's estimated to be ~10s KB size increase in OSS runtime build. I assume it's fine because this increase would be there anyway for the transition period.

Reply:
I removed those files from there; and moved to separate BUCK/bazel targets instead. This way we can choose to include them as deps when we roll it out.